examples/ray_serve/train_model.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import mlflow
+
+from sklearn.datasets import load_iris
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.metrics import mean_squared_error
+
+
+# Enable autologging
+mlflow.sklearn.autolog()
+
+# Load data
+iris_dataset = load_iris()
+data, target, target_names = (
+    iris_dataset["data"],
+    iris_dataset["target"],
+    iris_dataset["target_names"],
+)
+
+# Instantiate model
+model = GradientBoostingClassifier()
+
+# Split training and validation data
+np.random.shuffle(data)
+np.random.shuffle(target)
```

Comment:
Should we avoid shuffling `data` and `target` separately?

Reply:
I think we can use https://scikit-learn.org/stable/modules/generated/sklearn.utils.shuffle.html